app/modules/auth/Auth.tsx

```diff
@@ -5,18 +5,29 @@ import names from 'classnames'
 import cl from './Auth.module.scss'
 import { AuthInputs } from './types/auth.types'
 import { User } from 'firebase/auth'
+import { useRouter } from 'next/navigation'
+import { LOCAL_STORAGE_USER_ID_KEY } from './constants'
 
 type Props = {
   onSubmit: ({ email, password }: AuthInputs) => Promise<undefined | User>
 }
 
 const Auth = ({ onSubmit }: Props) => {
+  const router = useRouter()
+
   const { register, handleSubmit } = useForm<AuthInputs>()
 
   const onSubmitHandler: SubmitHandler<AuthInputs> = async (data) => {
     const user = await onSubmit(data)
-    console.log(user)
+
+    if (user) {
+      localStorage.setItem(LOCAL_STORAGE_USER_ID_KEY, user.uid)
```

Comment:
Better to create service in which we will do some action with local storage, this will increase flexibility of our application.